action.yml

```diff
@@ -1,66 +1,49 @@
 # Copyright 2020 The MathWorks, Inc.
 
-name: Run MATLAB 
-Tests description: >-
-  Run all tests in a MATLAB project and generate test artifacts. MATLAB includes
-  any files in your project that have a Test label. If your pipeline does not
-  leverage a MATLAB project, then MATLAB includes all tests in the the root of
-  your repository including its subfolders.
+name: Run MATLAB Tests
+description: >-
+  Run MATLAB and Simulink tests and generate artifacts
 inputs:
   source-folder:
     description: >-
-      Source code folder Location of the folder containing source code, relative
-      to the project root folder. The specified folder and its subfolders are
-      added to the top of the MATLAB search path. To generate a code coverage
-      report, MATLAB uses only the source code in the specified folder and its
-      subfolders. You can specify multiple folders using a colon-separated or a
-      semicolon-separated list.
-    required: false 
+      Location of the folder containing source code
+    required: false
     default: ""
   select-by-folder:
     description: >-
-      Locations of the folders used to select test suite elements, relative to
-      the project root folder. To create a test suite, MATLAB uses only the
-      tests in the specified folders and their subfolders. You can specify
-      multiple folders using a colon-separated or semicolon-separated list.
-    required: false 
+      Location of the folder containing test files
+    required: false
     default: ""
   select-by-tag:
     description: >-
-      Test tag used to select test suite elements. To create a test suite,
-      MATLAB uses only the test elements with the specified tag.
-    required: false 
+      Test tag used to select test suite elements
+    required: false
     default: ""
   test-results-pdf:
     description: >-
-      Path to write test results report in PDF format (currently not supported
-      on macOS platforms).
-    required: false 
+      Path to write test results report in PDF format (not supported on macOS)
```

Comment:
It might be good to remove the parenthetic info here as well. They can find it in the doc.

Reply:
Done! 😄